bayesian/generate_manifest.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from lxml.etree import SubElement, Element, ElementTree
+from io import BytesIO
+
+
+class PomXMLTemplate:
+
+    def __init__(self, json_data=None):
+        if json_data:
+            self._data = dict(json_data)
```

Comment:
Why there is casting to dict? Also if you request `json_data` to be present do not assign default value in constructor.